app/api/tutorials_api.rb

```diff
@@ -9,6 +9,8 @@ class TutorialsApi < Grape::API
       authenticated?
     end
 
+    ## --- Deprecated ---
```

Comment:
Dont we still need the ability to edit/add/etc individual activities/tutorials?
Doesn't this just change to include additional data associated with which activity set the activity is part of?